lesson 3/index.html

```diff
@@ -0,0 +1,151 @@
+        <td>Defines a table</td>
+    </tr>
+    <tr>
+        <td>&lttr&gt</td>
+        <td>Defines a row in a table</td>
+    </tr>
+    <tr>
+        <td>&ltth&gt</td>
+        <td rowspan="2">Defines a "header cell"/"cell" in a table</td>
+    </tr>
+    <tr>
+        <td>&lttd&gt</td>
+
+    </tr>
+    <tr>
+        <td colspan="2">Rarely used</td>
+
+    </tr>
+    <tr>
+        <td>&ltcaption&gt</td>
+        <td>Defines a table caption</td>
+    </tr>
+    <tr>
+        <td>&ltthead&gt</td>
+        <td>Groups the header content in a table</td>
+    </tr>
+    <tr>
+        <td>&lttbody&gt</td>
+        <td>Groups the body content in a table</td>
+    </tr>
+    <tr>
+        <td>&lttfoot&gt</td>
+        <td>Groups the footer content in a table</td>
+    </tr>
+    </tbody>
+
+</table>
+<hr>
+
+<form>
+    <p>
+        <label for="name">Name</label><br>
+        <input id="name" name = "name" type="text" placeholder="Name"/><br>
+    </p>
+    <p>
+        <label for="e-mail">E-mail</label><br>
+        <input id="e-mail" name = "e-mail" type="email"/><br>
+    </p>
+    <p>
+        <label for="Password">Password</label><br>
+        <input id="Password" name = "Password"/>
+    </p>
+
+    <p>
+        <label for="programming">Your programming level</label><br>
+        <input id="programming" name="level" type="range" min="0" max="10" value="5">
+    </p>
+
+    <p>
+        What does HTML stand for?<br>
+        <input id="I" name="HTML abbreviation" type="checkbox" value="1">
+        <label for="I">Hyperlinks and Text Markup Language</label><br>
+        <input id="II" name="HTML abbreviation" type="checkbox" value="2">
+        <label for="II">Home Tool Markup Language</label><br>
+        <input id="III" name="HTML abbreviation" type="checkbox" value="3">
+        <label for="III">Hyper Text Markup Language</label><br>
+    </p>
+
+    <p>
+        What does HTTP stand for?<br>
+        <input id="a" name="HTTP abbreviation" type="radio" value="1">
+        <label for="a">Hyper Text Translate Protocol</label><br>
+        <input id="b" name="HTTP abbreviation" type="radio" value="2">
+        <label for="b">Hyper Text Transmission Protocol</label><br>
+        <input id="c" name="HTTP abbreviation" type="radio" value="3">
+        <label for="c">Hyper Text Transfer Protocol</label><br>
+    </p>
+
+    <p>
+        <label for="comment">Comment</label><br>
+        <textarea id="comment" cols="25" rows="5" placeholder="Comment"></textarea>
+    </p>
+
+    <p>
+        <label for="abbreviation">What does HTML stand for?</label><br>
+        <select id="abbreviation" name ="abbreviation">
+            <option disabled selected>Your answer</option>
+            <option value="1">Hyperlinks and Text Markup Language</option>
+            <option value="2">Home Tool Markup Language</option>
+            <option value="3">Hyper Text Markup Language</option>
+        </select>
+    </p>
+
+    <p><button>Send</button></p>
```

Comment:
it's better always write type of button for more predictable behavior
`type="submit"`